src/org/labkey/test/components/ui/domainproperties/EntityTypeDesigner.java

```diff
@@ -81,6 +81,12 @@ public T setName(String name)
         return getThis();
     }
 
+    public boolean isNameFieldEnabled()
+    {
+        String disabledValue = Locator.tagWithId("input", "entity-name").findWhenNeeded(getDriver()).getAttribute("disabled");
+        return (null == disabledValue) || (!disabledValue.equalsIgnoreCase("true"));
```

Comment:
I think you could just do this:
`return Locator.tagWithId("input", "entity-name").findWhenNeeded(getDriver()).isEnabled();`